secure_uninstall/README.md

```diff
@@ -0,0 +1,5 @@
+Secure Uninstall
+================
+
+Ask admin password ('admin_passwd' key from config file)
+before to proceed to module uninstallation
```

Comment:
Admin db password might be known of the customer at a time or another of the project, perhaps for a short time for a ponctual action
Config file password will stay secret of all the users.
Could you have a look to https://github.com/OCA/server-tools/pull/171/files#diff-8600e5eba033357ad3d6fdc709908ccdR29, please.